packages/main/cypress/specs/DateRangePicker.cy.tsx

```diff
@@ -0,0 +1,570 @@
+import "../../dist/Assets.js";
+import { setLanguage } from "@ui5/webcomponents-base/dist/config/Language.js";
+import DateRangePicker from "../../src/DateRangePicker.js";
+
+describe("DateRangePicker general interaction", () => {
+	afterEach(() => {
+		// eslint-disable-next-line
+		cy.wait(200);
+	});
+	it("Custom Validation Error", () => {
+		cy.mount(<DateRangePicker formatPattern="dd/MM/yyyy"></DateRangePicker>);
+
+		cy.get<DateRangePicker>("[ui5-daterange-picker]")
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.realClick()
```

Comment:
i think it will be good to ensure that the input is focused after the click and before we type